pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -44,6 +47,9 @@ rustPlatform.buildRustPackage rec {
     gsettings-desktop-schemas
     libxkbcommon
     openssl
+    luajit
+    lua51Packages.lgi
```

Comment:
LGI is a runtime dep, not a build input.

Reply:
Added it to `runtimeDeps` in https://github.com/NixOS/nixpkgs/pull/307726/commits/9ff1da6d1de964ce27e11f944b68e36e0097805b